### PR TITLE
Fixes #270815 - Alon was a bachelor

### PR DESCRIPTION
**Pull request by orrabin**
מזל טוב!!!!

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+    X@.    /.,   ~@~    ~@~   \|           H
+    H      /  '"*.,,*"'*,.,*'"\|         `@H
+   @X@~   /                    \          X@
+  ~ %@,  /                      \        ,@H ~
+    H   /                        \         H
+    H@.@~                       ~@\        %@,
+  ,@X `'"*'*,  ~@~     ~@~  ,.*'"*"      ~@X
+    H@~      '"*,.*"'"*.,*"'               H@.
+    H><>gpyy<><><><><><><><><><><><><><><><H
+
```
Comment (orrabin) [inline Gemfile:34]: Mazal Tov!!!

Comment (unorthodoxgeek) [inline Gemfile:34]: מזל טוב אלון!

Comment (shlomizadok) [inline Gemfile:34]: מזל טוב אלון!

Comment (tbrisker) [inline Gemfile:34]: Mazal Tov!

Comment (dLobatog) [inline Gemfile:34]: Mazal tov!

Comment (lzap) [inline Gemfile:34]: What a bride! And gent he is. Congrats!

Comment (isratrade) [inline Gemfile:34]: Congratulations on your last day of single life and first day of married life!!!

Comment (amirfefer) [inline Gemfile:34]: Mazal Tov!
And remember... Happy wife happy life (:

Comment (ares) [inline Gemfile:34]: Mazal tov! Trust me, the game is not over, it starts now!

Comment (jlsherrill) [inline Gemfile:34]: Congrats!

**Comment by ShimShtein** [comment]
Closed as [wedding_party](#zzz)
Mazal tov Alon!!!!

File: Gemfile (hunk 1 above)
Comment (ShimShtein) [inline Gemfile:34]: Closed as [wedding_party](#zzz)
Mazal tov Alon!!!!

Comment (alongoldboim) [inline Gemfile:34]: Thanks guys! it was great.

**Pull request closed without merging**